changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "5db23135-2470-428b-98ab-dd33f3c8a5cf",
+          "ApiChange": "Member",
+          "ChangedApiName": "customPasswordAddition",
+          "ChangeType": "Addition",
+          "Description": "Added the [customPasswordAddition](https://docs.microsoft.com/en-us/graph/api/resources/passwordcredentialconfiguration?view=graph-rest-beta) enum to passwordCredentialConfiguration",
```

Seems there's a bug in the tooling. The description generated wasn't completely accurate.
```suggestion
          "Description": "Added the `customPasswordAddition` member to the **appCredentialRestrictionType** enumeration. Use this new restriction for the [application authentication methods policy](https://docs.microsoft.com/en-us/graph/api/resources/applicationauthenticationmethodpolicy?view=graph-rest-beta) to restrict all new custom (non-Azure AD generated) password secrets on applications created after 01/01/2015.",
```

---

@FaithOmbongi , with the date in the description a user might form an opinion that we are restricting creds on apps created after 2015. I think we should remove the date.

---

Updated the verbiage a bit. I think we should be good to go.